R/pkg/R/generics.R

```diff
@@ -549,6 +549,10 @@ setGeneric("write.parquet", function(x, path) { standardGeneric("write.parquet")
 #' @export
 setGeneric("saveAsParquetFile", function(x, path) { standardGeneric("saveAsParquetFile") })
 
+#' @rdname write.text
+#' @export
+setGeneric("write.text", function(x, path) { standardGeneric("write.text") })
+
```

Comment:
I found all functions in SQLContext.R did not use ```setGeneric```, is it on purpose or a bug?
If it's a bug, I can fix it.